library/Zend/Validate/EmailAddress.php

```diff
@@ -453,7 +453,7 @@ private function _validateMXRecords()
 
         //decode IDN domain name if possible
         if (function_exists('idn_to_ascii')) {
-            $hostname = idn_to_ascii($this->_hostname);
+            $hostname = idn_to_ascii($this->_hostname, 0, INTL_IDNA_VARIANT_UTS46);
```

Comment:
There was a test throwing a deprecation notice (but not failing) because of this.

The default constant used to this function (if not specified) is `INTL_IDNA_VARIANT_2003` which was deprecated in php 7.2.0.

>INTL_IDNA_VARIANT_2003 has been deprecated; use INTL_IDNA_VARIANT_UTS46 instead. 

The `INTL_IDNA_VARIANT_UTS46` constant was added in php 5.4.  So this will break now for php 5.4 (but the lowest we're testing against is 5.6 since everything else is EOL).